docs/git-commands.md

```diff
@@ -0,0 +1,147 @@
+# Git Command Guide
+The following is a number of scenarios that you will encounter while contributing to this repo. A convention in this document is to use `<< >>` to indicate text that you will need to replace when running the commands on your own machine.
+
+## Clone this REPO
+Before you begin, you will need to clone this repo
+```
+git clone https://github.com/MountSionCBSSecondary/SchoolWebsite.git
+```
+The above command will download a copy of the repo to your local machine. This copy of the repo will have the remote name of `origin`.
+
+
+## Add your own Remote
+Once you have the origin remote you will need to add a remote to your own fork of the repo. This can be done with the following command, just replace the placeholder with your own github name.
+A remote is a URL and it is github's way of specifying where a repo is stored. There is also two types of remotes `https` and `ssh` For example the school website `https` remote is `https://github.com/MountSionCBSSecondary/SchoolWebsite.git` and the `ssh` remote is `git@github.com:MountSionCBSSecondary/SchoolWebsite.git` 
+
+It should also be noted you will need to have forked the repo, how to do this can be seen in [our tutorial](Tutorial.md)
```

Comment:
Can we put this as a note first?

Reply:
Yeah, I will leave it noted it, and also add a note and link to the top of the document about the tutorial page